docs/creating-new-project.md

```diff
@@ -15,7 +15,12 @@
 1. Customize *blt/blt.yml* if desired, such as to choose an install profile.
 
     By default, BLT will install sites using the [*lightning*](https://github.com/acquia/lightning) profile. You can change this to any other core, contributed, or custom profile in your codebase. Make sure to download the profile if necessary, e.g., `composer require acquia/headless_lightning:~1.1.0`.
-
+    
+    Make sure to also update blt.yml with the machine name of your desired profile:
```

Comment:
This kind of sounds like they _have_ to update it. But they only have to update it if they don't want to use Lightning.

Reply:
What if we changed it to something like, 
> To use a profile other than Lightning, enter the name of the profile in `blt/blt.yml` in the `profile:name` setting. For example...